bucket/cv4pve-pepper.json

```diff
@@ -0,0 +1,29 @@
+{
+    "version": "1.5.0",
+    "description": "Launching SPICE remote-viewer to access VM running on Proxmox VE.",
+    "homepage": "https://github.com/Corsinvest/cv4pve-pepper",
```

Comment:
Why not https://www.cv4pve-tools.com/?

Reply:
Ok, I have changed the homepage to https://www.cv4pve-tools.com/.